keyboards/ymdk/ymd60/keymaps/via/keymap.c

```diff
@@ -0,0 +1,51 @@
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum layer_names {
+    _BASE,
+    _FN1,
+    _FN2,
+    _FN3
+};
+
+// enum layer_keycodes { };
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*  Row:    0        1        2        3        4        5        6        7        8        9         10       11        12       13       14     */
+    [_BASE] = LAYOUT(
+                KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,     KC_0,    KC_MINS,  KC_EQL,  KC_BSPC, KC_DEL,
+                KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,     KC_P,    KC_LBRC,  KC_RBRC, KC_BSLS,
+                KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,     KC_SCLN, KC_QUOT,           KC_ENT,           
+                KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM,  KC_DOT,  KC_SLSH,           KC_RSFT, MO(_FN1),            
+                KC_LCTL, KC_LGUI, KC_LALT,                                     KC_SPC,                     KC_RALT,           KC_RGUI, KC_RCTL
+            ),
+
+    /*  Row:    0        1        2        3        4        5        6        7        8        9         10       11        12       13       14     */
+    [_FN1] = LAYOUT(
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,  _______, _______,
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______,  _______,
+                _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______,           _______,           
+                _______,          _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______,           _______, _______,            
+                _______, _______, _______,                                     _______,                    _______,          _______,  _______
```

Comment:
Again, providing a default HHKB-like keymap would be useful here; otherwise a layer which is completely empty should be removed (together with the `MO(1)` reference to it).

Reply:
I will add some useful button into this layer (like `QK_BOOT` and RGBs, thank you :D